deploy/dist-test/upload.sh

```diff
@@ -24,6 +24,8 @@ docker-compose \
     cpp \
     inv func.user demo \
     func.upload-user demo \
+    func mpi mpi_bcast \
+    func.upload mpi mpi_bcast \
```

Comment:
If it's not too inefficient it's probably easier just to do the whole `mpi` user as it would be nice to add more distributed MPI tests in future.